docs/backends/_utils.py

```diff
@@ -36,7 +36,11 @@ def find_member_with_docstring(member):
         return member
 
     cls = member.parent
-    for base in cls.resolved_bases:
+    resolved_bases = cls.resolved_bases
+    # If we're a SQLBackend (likely) then also search through to `BaseBackend``
+    if (sqlbackend := resolved_bases[0]).name == "SQLBackend":
```

Comment:
Is there a more generic fix for this, like altering the lookup code to traverse the class hierarchy?